[DOC-306] [DOC-320] [DOC-357] [yba] 2024.1 Release management enhancement, 2024.1 version naming changes. (#22173)
yugabyte/yugabyte-db: docs/content/preview/yugabyte-platform/manage-deployments/ybdb-releases.md

---
title: Manage YugabyteDB releases available in YugabyteDB Anywhere
headerTitle: Manage YugabyteDB releases
linkTitle: Manage releases
description: Use YugabyteDB Anywhere to upgrade the YugabyteDB software on universes.
headcontent: View and import YugabyteDB releases
menu:
  preview_yugabyte-platform:
    identifier: ybdb-releases
    parent: upgrade-software
    weight: 15
type: docs
---

> Before you can upgrade your universe to a specific version of YugabyteDB, verify that the release is available and, if necessary, import the release into YugabyteDB Anywhere.

Reviewer: ```suggestion
Before you can upgrade your universe to a specific version of YugabyteDB, verify that the release is available in YugabyteDB Anywhere and, if necessary, import it .
```

To view the YugabyteDB releases that are available in your YugabyteDB Anywhere instance, do the following:

- Click the user profile icon and choose **Releases**.

    ![Releases](/images/yp/releases-list-2024.png)

You can filter the list by text and release type, and choose to display only allowed releases.

To view the release details, select a release in the list.

To delete or disable a release, click its corresponding **Actions**. Disabled releases are not available when creating universes.

## Import a release

> If a release that you want to install on a universe is not available, import it as follows:

Reviewer: ```suggestion
If a release that you want to install or upgrade is not available, import it as follows:
```

1. On the **Releases** page, click **New Release** to open the **Import Database Release** dialog as shown in the following illustration:

>     ![Import Release](/images/yp/import-releases-2024.png)

Reviewer: The image has a lot of white space. Can we trim it?
